packages/discord.js-utilities/src/lib/PaginatedMessage.ts

```diff
@@ -300,7 +313,9 @@ export class PaginatedMessage {
 
 		// Do not remove reactions if the message, channel, or guild, was deleted:
 		if (this.response && !PaginatedMessage.deletionStopReasons.includes(reason)) {
-			await this.response.reactions.removeAll();
+			if ((this.response.channel as TextChannel).permissionsFor(this.response.client.user?.id || '')?.has('MANAGE_MESSAGES')) {
+				await this.response.reactions.removeAll();
+			}
```

Comment:
Similar change as above. Don't typecast, that's extremely unsafe and this will crash in DMs because `permissionsFor` is not a method on `DMChannel`

Reply:
done